apps/gallery/index.html

```diff
@@ -212,11 +212,11 @@ <h1 id="edit-title" data-l10n-id="edit"></h1>
       </section>
 
       <!-- buttons for selecting the type of edit to perform -->
-      <footer id="edit-toolbar">
-        <a id="edit-exposure-button" class="button" data-icon="brightness"></a>
-        <a id="edit-crop-button" class="button" data-icon="crop"></a>
-        <a id="edit-effect-button" class="button"></a>
-        <a id="edit-enhance-button" class="button"></a>
+      <footer id="edit-toolbar" role="toolbar">
+        <a id="edit-exposure-button" class="button" data-icon="brightness" data-l10n-id="exposure-button" role="button"></a>
+        <a id="edit-crop-button" class="button" data-icon="crop" data-l10n-id="crop-button" role="button"></a>
+        <a id="edit-effect-button" class="button" data-l10n-id="effect-button" role="button"></a>
+        <a id="edit-enhance-button" class="button"  data-l10n-id="enhance-button" role="button" aria-pressed="false"></a>
```

Comment:
Similar comment as bug 1068984 patch, If simple consider changing links to button element.